quantum/config_common.h

```diff
@@ -19,6 +19,8 @@
 /* diode directions */
 #define COL2ROW 0
 #define ROW2COL 1
+#define EITHERWAY 2 /* Compensation for diode direction dissonance */
+#define BOTHWAYS 3 /* Duplex-Matrix */
```

Comment:
IMO we should keep the existing naming convention here, so `COL2ROW2COL` and `ROW2COL2ROW`. The documentation can explain what these mean, as they do already for `COL2ROW` and `ROW2COL`.

Reply:
Well, I find ...
`#define DIODE_DIRECTION BOTHWAYS`
`#define DIODE_DIRECTION EITHERWAS`
nifty. But it's only a name. I'll change it to what ever we agree on.
That said, ROW2COL2ROW does not convey the working of EITHERWAY. Please come up with a good name.
Anyone?

btw, COL2ROW2COL was baptized by @drashna in https://github.com/qmk/qmk_firmware/pull/6313#issuecomment-511488976 .